code/api/src/migrations/7-user_products.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
```

Comment:
This is the migration for UserProducts joins table, has kept which determines if products is kept

Reply:
This is awesome. So nice to see something that is readable!

Reply:
Thanks for getting this done!